src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -250,7 +250,7 @@ protected final void verify(Checker checker,
             for (int index = 0; index < expected.length; index++) {
                 final String expectedResult = messageFileName + ":" + expected[index];
                 final String actual = lnr.readLine();
-                assertWithMessage("error message %s", index)
+                assertWithMessage("error message %s does not match expected message", index)
```

Comment:
Error Message at position %s of expected, does .....

Reply:
done.